src/propagatorschnittstelle.f90

```diff
@@ -1378,7 +1391,7 @@ function OPI_Plugin_propagate(propagator, data, julian_day, dt) result(opi_error
           end if
 
           !** allocate the ephemeris array
-          allocate(ephemeris(1000000, 1:28, data_size)) !** the 1000000 is the maximum allowed at the moment. It is fixed to avoid errors with PICARD. The 28 is: 1 date, 6 state, 21 co-variances
+          allocate(ephemeris(1000000, 1:43, data_size)) !** the 1000000 is the maximum allowed at the moment. It is fixed to avoid errors with PICARD. The 28 is: 1 date, 6 state, 21 co-variances
```

Comment:
Update the comment with the range of values in the second dimension of the array. Also you can remove the part about PICARD.

Reply:
Done